devo/sender/data.py

```diff
@@ -90,6 +99,7 @@ def __init__(self, address=None):
             self.address = address
             self.hostname = socket.gethostname()
             self.sec_level = None
+            self.verify_config = False
```

Comment:
Test `test_tcp_rt_send` fails without it, this happens because `verify_config` is part of the constructor for `Sender` so if `verify_config` is null it will break, I will change it so if `verify_config` is null it will not do the verification.